src/hotspot/cpu/riscv/upcallLinker_riscv.cpp

```diff
@@ -27,12 +27,12 @@
 #include "prims/universalUpcallHandler.hpp"
 #include "utilities/debug.hpp"
 
-address ProgrammableUpcallHandler::generate_optimized_upcall_stub(jobject receiver, Method* entry,
-                                                                  BasicType* in_sig_bt, int total_in_args,
-                                                                  BasicType* out_sig_bt, int total_out_args,
-                                                                  BasicType ret_type,
-                                                                  jobject jabi, jobject jconv,
-                                                                  bool needs_return_buffer, int ret_buf_size) {
+address UpcallLinker::generate_optimized_upcall_stub(jobject receiver, Method* entry,
```

Comment:
Hasn't "generated_optimized_upcall_stub" changed too?

Reply:
Right, I missed that one. Good catch!